src/main/kotlin/org/stt/cli/ReportPrinter.kt

```diff
@@ -153,17 +155,17 @@ constructor(private val queries: TimeTrackingItemQueries,
 
             var worktimeDuration = Duration.ZERO
             var breakTimeDuration = Duration.ZERO
-            for ((duration, comment) in reportingItems) {
+            for (reportItem in reportingItems) {
```

Comment:
This change just seems to add noise, why?

Reply:
hmm - I guess because I didn't knew the feature of "destructing declaration" within for-loops well. and I thought, that I need to other (new) attributes here ... but as me is never using the CLI-mode...

I guess `for ((duration, roundedDuration, comment) in reportingItems)` will do the trick, then `roundedDuration` is unused (also smelly)

Reply:
@Bytekeeper wdyt?

Reply:
Well, you can just replace it with a _ aka:
(duration, _, comment)

Reply:
Wow - thank you - a new learning 😀